package.json

```diff
@@ -42,7 +46,23 @@
     "sinon": "^2.1.0"
   },
   "scripts": {
-    "test": "mocha --require coffee-script/register --compilers coffee:coffee-script --compilers js:babel-core/register test/*.js",
-    "watch:test": "npm t -- --watch"
+    "test": "xo --quiet && mocha --require coffee-script/register --compilers coffee:coffee-script --compilers js:babel-core/register test/*.js",
+    "watch:test": "npm t -- --watch",
+    "lint": "xo --quiet"
+  },
+  "xo": {
+    "space": true,
+    "semicolon": false,
+    "rules": {
+      "no-var": "warn",
+      "no-use-before-define": 1,
+      "no-await-in-loop": 1,
+      "import/prefer-default-export": 1,
+      "no-negated-condition": 1,
+      "guard-for-in": 1
```

Comment:
```
"no-use-before-define": 1,
"no-await-in-loop": 1,
"import/prefer-default-export": 1,
"no-negated-condition": 1,
"guard-for-in": 1
```
@Mikanebu could you please explain what this cinfigurations do?

Reply:
same here, from `datahub-cli`

Reply:
Same here - this is not `datahub-cli`. `datahub-cli` has its styling convention depending on its needs. 

We may need them, just would be useful to know what they do so we are sure how to code.

Reply:
@zelima will be removed

Reply:
`no-use-before-define` it happens when we call function before the declaring it, in `messages.js` it calls first `formatGist` then declares it later

Reply:
OK great thanks, Now it is clear. So:
* remove `no-use-before-define` and move `formatGist` at the top or `sendGist` at the bottom, and it will not error
* remove for now `no-await-in-loop ` 
* remove `import/prefer-default-export `
* remove `no-negated-condition `
* keep `guard-for-in` if it complains